### PR TITLE
Remove obsolete members from Avalonia.Base namespace

### PR DESCRIPTION
**Pull request by Takoooooo**
Though there are two places where i have no idea what to do.
1) https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Base/Media/DrawingContext.cs#L73-L74
2) https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Base/VisualTree/IVisualWithRoundRectClip.cs

## Fixed issues
Part of https://github.com/AvaloniaUI/Avalonia/issues/6666

### CODE REVIEW

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `0.10.999-cibuild0023020-beta`. (feed url: https://nuget.avaloniaui.net/repository/avalonia-all/index.json) [PRBUILDID]

File: src/Avalonia.Base/Threading/DispatcherPriority.cs [hunk 1]
```diff
@@ -22,21 +22,6 @@ private DispatcherPriority(int value)
         /// </summary>
         public static readonly DispatcherPriority MinValue = new(0);
 
-        /// <summary>
-        /// The job will be processed when the system is idle.
-        /// </summary>
-        [Obsolete("WPF compatibility")] public static readonly DispatcherPriority SystemIdle = MinValue;
```
Comment (kekekeks) [inline src/Avalonia.Base/Threading/DispatcherPriority.cs:28]: We need a way to mark properties that are here to lighten the burden of porting WPF code, but shouldn't be used for the new code

Comment (Takoooooo) [reply src/Avalonia.Base/Threading/DispatcherPriority.cs:28]: My suggestion is just to revert this change and leave this as is. Currently i can't think of a better solution

Comment (maxkatz6) [reply src/Avalonia.Base/Threading/DispatcherPriority.cs:28]: What about Obsolete + EditorBrowsable(Never)?

Comment (Takoooooo) [reply src/Avalonia.Base/Threading/DispatcherPriority.cs:28]: > What about Obsolete + EditorBrowsable(Never)?

If we want our users to use these fields when porting from WPF then i think EditorBrowsable isn't suitable here.

Comment (maxkatz6) [reply src/Avalonia.Base/Threading/DispatcherPriority.cs:28]: > My suggestion is just to revert this change and leave this as is

Let's do it @Takoooooo

Comment (Takoooooo) [reply src/Avalonia.Base/Threading/DispatcherPriority.cs]: @maxkatz6 Done

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `0.10.999-cibuild0023193-beta`. (feed url: https://nuget.avaloniaui.net/repository/avalonia-all/index.json) [PRBUILDID]

**Review by maxkatz6 — APPROVED** [review]

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `0.10.999-cibuild0023195-beta`. (feed url: https://nuget.avaloniaui.net/repository/avalonia-all/index.json) [PRBUILDID]

**Pull request merged**